test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities

```

Comment:
@istoimenova Agree with Alexandr. For CRQ coverage enough part of cases from list.
Additional for Alexandr's comment I would add 010, 019, 031, 033, 034, 035, 036, 037, 046 - 058 I think also applicable for CRQ.
Other tests out of scope CRQ coverage. But usefull for testing json members (which not tested now). But it is another scope of work. I propose separete overhead tests or mark them in some way to decrease influence of defects which will be found by this tests in scope of CRQ testing.